src/lib/tsconfig.lib.prod.json

```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "./tsconfig.lib.json",
-  "angularCompilerOptions": {
-    "enableIvy": false
-  }
+  "extends": "./tsconfig.lib.json"
```

Comment:
I needed to change this in order for the production build to complete.  Otherwise, [this](https://stackoverflow.com/questions/60234048/angular-9-library-publish-error-trying-to-publish-a-package-that-has-been-compi) error was thrown.  I am not seeing it in your build currently, and I'm not remembering if it was during the release process at the moment.

Reply:
`"compilationMode": "partial"` can be used in this case

Reply:
https://stackoverflow.com/a/68197875/6464572